pkg/kubernetes/version.go

```diff
@@ -33,12 +34,14 @@ func (c *controller) GetCurrentVersion(ul *unstructured.UnstructuredList, kind,
 
 	// Filter out all unassociated objects based on the moniker.spinnaker.io/cluster annotation.
 	manifestFilter := NewManifestFilter(ul.Items)
-	lastIndex := strings.LastIndex(name, "-v")
 
-	if lastIndex != -1 {
-		cluster = kind + " " + name[:lastIndex]
-	} else {
+	re := regexp.MustCompile(`(.*)-v(\d){3}`)
```

Comment:
Shouldn't this be `(.*)-v(\d){3}$`. Otherwise, something like service-v000-myextension would match.

Reply:
Yes, that's right. Thanks!